pkg/systemd/overview-cards/realmd.jsx

```diff
@@ -467,7 +467,7 @@ export const RealmButton = ({ realmd_client }) => {
                     excuse={ buttonTooltip }>
             <Button id="system_information_domain_button" variant="link"
                     onClick={onClicked}
-                    isInline isDisabled={buttonDisabled} aria-label={_("Join domain")}>
+                    isInline isDisabled={buttonDisabled} aria-label={_("Install Domain Support")}>
```

Comment:
This change does not really change the Button text, it only changes the [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) so this wouldn't lead to any visible changes.

What should happen is that the `buttonText` is changed according to the state of `realmd_client.install_realmd`. If it's not installed we should show a `Install realmd support` button.


Note that this also might require tests to be changed

```
test/verify/check-system-realms:                b.wait_text(self.domain_sel, "Join domain")
test/verify/check-system-realms:                b.wait_text_not(self.domain_sel, "Join domain")
test/verify/check-system-realms:        b.wait_text(self.domain_sel, "Join domain")
test/verify/check-system-realms:        b.wait_text(self.domain_sel, "Join domain")
test/verify/check-system-realms:        b.wait_text(self.domain_sel, "Join domain")
test/verify/check-system-realms:        b.wait_text(self.domain_sel, "Join domain")
```

Reply:
@jelly do all the specified tests need to be updated to "Install realmd support" ?